eip712/messages.py

```diff
@@ -144,3 +146,18 @@ def signable_message(self) -> SignableMessage:
             HexBytes(hash_domain(self._domain_)),
             HexBytes(hash_eip712_message(self._body_)),
         )
+
+
+def encode(typed_data: Dict[str, Any]) -> List[bytes]:
+    parts = [bytes.fromhex("1901"), hash_domain(typed_data)]
+
+    if typed_data["primaryType"] != "EIP712Domain":
+        parts.append(hash_eip712_message(typed_data))
+
+    return parts
+
+
+def calculate_hash(typed_data) -> str:
```

Comment:
ok i think i got this pretty simple now, and it only used methods that were already imported!

Reply:
realizing we have an issue with `.signable_message` then.